microsoft-edge/devtools-guide-chromium/sources/opening-sources-in-vscode.md

```diff
@@ -22,7 +22,7 @@ First, select DevTools > **Settings** > **Experiments** > **Open source files in
 
 With this experiment enabled, suppose that in Microsoft Edge, you go to a local server (such as `http://localhost` or `http://127.0.0.1`), or open a local file.
 
-![Microsoft Edge with a local file open](./opening-sources-in-vscode-images/experiment-sources-in-code-local-project.msft..png)
+![Microsoft Edge with a local file open](./opening-sources-in-vscode-images/experiment-sources-in-code-local-project.png)
 
```

Comment:
```suggestion
<!-- todo:
instead of using 
https://github.com/codepo8/simple-to-do
use:
https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do
Rewrite article as numbered action steps -> results.
-->
```